classes/Tools.php

```diff
@@ -2881,13 +2881,14 @@ public static function generateRobotsFile($executeHook = false)
                 }
 
                 // Disallow multilang directories
-                if (!empty($languagesIsoIds)) {
+                if (!empty($languagesIsoIds) && count($languagesIsoIds) > 1) {
```

Comment:
```suggestion
                if (is_array($languagesIsoIds) && count($languagesIsoIds) > 1) {
```
The return of `getIsoIds` can be false, so the count will fail. This check can also be done above instead of redo it on each loop.

```php
$containsIsoIds = is_array($languagesIsoIds) && count($languagesIsoIds) > 1;
```